tests/backup/backup_helper.go

```diff
@@ -1218,8 +1227,8 @@ func restoreSuccessCheck(restoreName string, orgID string, retryDuration int, re
 		return "", true, fmt.Errorf("expected status of %s - [%s] or [%s], but got [%s]",
 			restoreName, api.RestoreInfo_StatusInfo_PartialSuccess.String(), api.RestoreInfo_StatusInfo_Success, restoreResponseStatus.GetStatus())
 	}
-	task.DoRetryWithTimeout(restoreSuccessCheck, time.Duration(retryDuration)*time.Minute, time.Duration(retryInterval)*time.Second)
-
+	_, err := task.DoRetryWithTimeout(restoreSuccessCheck, time.Duration(retryDuration)*time.Minute, time.Duration(retryInterval)*time.Second)
+	log.FailOnError(err, "Verifying status for restore-%s", restoreName)
```

Comment:
Same here. Should not be handling error inside the function

Reply:
done!!